### PR TITLE
Add suffixes to typos from "q" to "red"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -33568,30 +33599,42 @@ queiried->queried
...
+quesetions->questions
 quesets->quests
+quesion->question
+quesions->questions
 quesiton->question
 quesitonable->questionable
 quesitons->questions
 quesr->quest
+quesrion->question
+quesrions->questions
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:33622]: The -ed form could be added here as well.

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:33622]: Perhaps -ing too?

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:33622]: And questionnaire(s)?

Comment (int-y1) [reply codespell_lib/data/dictionary.txt]: Done

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**